src/client/pages/ConnectPayment/sections/ConnectPayment.tsx

```diff
@@ -5,105 +5,131 @@ import { Market, useMarket } from 'components/utils/CurrentLocale'
 import { AdyenCheckout } from 'pages/ConnectPayment/components/AdyenCheckout'
 import { TrustlyCheckout } from 'pages/ConnectPayment/components/TrustlyCheckout'
 import { useTextKeys } from 'utils/textKeys'
+import {
+  MOBILE_SCREEN_MEDIA_QUERY,
+  MEDIUM_SCREEN_MEDIA_QUERY,
+} from 'utils/mediaQueries'
 import { useVariation, Variation } from 'utils/hooks/useVariation'
 import { AVYWindow } from 'utils/tracking/signing'
 import { ErrorModal } from '../components/ErrorModal'
 
 const SITEWRAPPER = 1300
-const BP = 800
-const MOBILE = 450
 
-const Background = styled('div')({
-  position: 'fixed',
-  top: 0,
-  right: 0,
-  bottom: 0,
-  left: 0,
-  backgroundColor: colorsV3.gray900,
-  zIndex: -1,
-})
+const Background = styled.div`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: ${colorsV3.gray900};
+  z-index: -1;
+`
 
-const InnerWrapper = styled('div')({
-  display: 'flex',
-  flexDirection: 'row',
-  justifyContent: 'space-between',
-  alignItems: 'center',
-  maxWidth: SITEWRAPPER,
-  margin: 'auto',
-  paddingLeft: '10px',
-  paddingRight: '10px',
-  paddingTop: '25vh',
-  color: colorsV3.gray500,
-  [`@media (max-width: ${BP}px)`]: {
-    maxWidth: '100%',
-    flexDirection: 'column',
-    paddingTop: 40 + 70,
-  },
-})
+const InnerWrapper = styled.div`
+  display: flex;
+  width: 80%;
+  margin: 0 auto;
+  flex-direction: row;
+  align-items: flex-start;
+  max-width: ${SITEWRAPPER}px;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  padding-top: 7.25rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 6.75rem;
+    flex-direction: column-reverse;
+    width: 100%;
+  }
+`
 
-const TextColumn = styled('div')({
-  width: '60%',
-  paddingRight: 20,
-  paddingLeft: 20,
-  paddingBottom: 40,
+const TextColumn = styled.div`
+  width: 60%;
+  padding-bottom: 2.5rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+  }
+`
 
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-    textAlign: 'center',
+const ImageColumn = styled.div`
+  width: 40%;
+  margin-top: 0.5rem;
+  padding-left: 2rem;
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    width: 100%;
+    padding: 0;
+    text-align: center;
   },
-})
-
-const ImageColumn = styled('div')({
-  width: '40%',
-  paddingRight: 20,
-  paddingLeft: 20,
-  paddingBottom: 40,
+}
+`
 
-  [`@media (max-width: ${BP}px)`]: {
-    width: '100%',
-    textAlign: 'center',
-  },
-})
+const Header = styled.h1`
+  margin-top: 0;
+  margin-bottom: 2.25rem;
+  font-size: 3rem;
+  color: ${colorsV3.white};
+  ${MOBILE_SCREEN_MEDIA_QUERY} {
+    text-align: center;
+    margin-bottom: 2rem;
+    font-size: 2rem;
+    line-height: 2rem;
+  }
+`
```

Comment:
According to Figma, adjust the line-heights a bit
```suggestion
const Heading = styled.h1`
  margin-top: 0;
  margin-bottom: 2.25rem;
  font-size: 3rem;
  line-height: 3.5rem;
  color: ${colorsV3.white};
  ${MOBILE_SCREEN_MEDIA_QUERY} {
    text-align: center;
    margin-bottom: 2rem;
    font-size: 2rem;
    line-height: 2.5rem;
  }
`
```